### PR TITLE
Bump version build push release image

### PR DESCRIPTION
**Pull request by eceakcck**
[GE-663](https://toptal-core.atlassian.net/browse/GE-663)

### Description

To fix the warning on Blackfish-Frontend project, we updated the `build-push-release-image` job..

### How to test

- Tested through [this PR](https://github.com/toptal/blackfish-frontend/pull/822)
### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/davinci-github-actions/blob/master/_docs/contribution/changeset-guidelines.md) (if needed)

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

[GE-663]: https://toptal-core.atlassian.net/browse/GE-663?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .changeset/famous-apricots-live.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'davinci-github-actions': patch
+---
+
+---
+
+- updating internal packages for build-push-release-image
```
Comment (OndrejTuma) [inline .changeset/famous-apricots-live.md:7]: nit: for better changelog message
```suggestion
### build-push-release-image
- bump yarn-install action to latest
```

File: build-push-release-image/action.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: toptal/davinci-github-actions/yarn-install@v4.8.2
+    - uses: toptal/davinci-github-actions/yarn-install@v4.8.3
```
Comment (OndrejTuma) [inline build-push-release-image/action.yml:24]: you might use the latest version 🙇 
```suggestion
    - uses: toptal/davinci-github-actions/yarn-install@v4.8.5
```

**Review by OndrejTuma — APPROVED** [review]
Thanks for contribution, have only few nit comments 👌

**Review by mkrl — APPROVED** [review]
Thank you for contributing!

**Review by jalolk — APPROVED** [review]
@eceakcck 🚀

File: build-push-release-image/action.yml (hunk 2 above)
Comment (eceakcck) [reply build-push-release-image/action.yml:24]: To close this ticket, I'll move forward as is for now if it's ok :slightly_smiling_face:

**Pull request merged**